src/main/java/com/ldtteam/structurize/placement/BlueprintIterator.java

```diff
@@ -47,6 +48,19 @@
      */
     private boolean isRemoving;
 
+    /**
+     * Iteration version (1 = original, 2 = outside in).
+     */
+    private static int ITERATION_VERSION = Structurize.getConfig().getServer().iterationVersion.get();
+
+    /**
+     * Tracking current state.
+     */
+    private int min_x;
+    private int max_x;
+    private int min_z;
+    private int max_z;
```

Comment:
Perhaps consider using `MutableBoundingBox`?  Though that has Y as well.

Reply:
Yeah, that's why I didn't use any other.

Reply:
the position we expose should be immutable though

Reply:
He only means the tracking variables here